components/app/landing/LandingDaoProposals.vue

```diff
@@ -19,31 +19,21 @@
               </div>
             </div>
             <div class="mt-5">
-              <div class="flex">
-                <img class="mr-2" src="@/assets/images/icons/start-fund.svg" alt="funding-start">
-                <p class="small-text text-primary">Start</p>
-              </div>
-              <p class="small-text">{{project.startDate}}</p>
-            </div>
-            <div class="mt-3">
-              <div class="flex">
-                <img class="mr-2" src="@/assets/images/icons/finish-fund.svg" alt="funding-finish">
-                <p class="small-text text-primary">Finish</p>
-              </div>
-              <p class="small-text">{{project.endDate}}</p>
+              <p class="small-text">{{project.description | truncate(90)}}</p>
```

Comment:
thought (non-blocking): maybe we could extract this value into a global variable so that we can use it everywhere? We might need the same value for the descriptions for the "normal" projects, right?